index.js

```diff
@@ -1,6 +1,8 @@
 //https://github.com/angular/angular.js/pull/10732
 
-var angular = require('angular');
+if (typeof angular !== 'undefined') {
```

Comment:
And hoisting in this case would look like when interpretation be `3th` line `{...window.angular: undefined}` or a function if it was required early.